src/styles/main.scss

```
@@ -1,3 +1,66 @@
-body {
+/* #region roots */
+
+html,
+body,
+div,
+section,
+::after,
+::before {
+  padding: 0;
   margin: 0;
 }
+
+/* #endregion */
+
+@import "./variables";
+// november wed 30
+.calendar {
+  margin: 50px auto;
+  display: flex;
+  width: $main-width-calendar;
+  gap: 1px;
+  flex-wrap: wrap;
+
+  &__days {
+    display: none;
+    width: $main-width-day;
+    height: $main-width-day;
+    box-sizing: border-box;
+    border: 1px solid black;
+    background-color: #eee;
+    position: relative;
+    cursor: pointer;
+    transition: all 0.5s ease;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: rgb(254, 191, 203);
+    }
+
+    @for $dat from 1 through 31 {
+      &:nth-child(#{$dat}) {
+        &::before {
+          display: block;
+          text-align: center;
+          line-height: $main-line-height-day;
+          content: "#{$dat}";
+          font-size: 30px;
+          font-family: Arial, Helvetica, sans-serif;
```

```suggestion
          content: "#{$dat}";
```
There is no need to generate all these styles in the cycle, consider generating content here and moving other styles outside the cycle.